src/server/api/controllers/events.js

```diff
@@ -77,7 +84,7 @@ export default class EventsController {
       .then((response) => response.json())
       .then((events) => {
         res.send({ list: events.data.allEvents.sort((a, b) =>
```

Comment:
I think that's a bit of premature optimization - this is a client and it knows what it just requested